sdk/communication/AzureCommunication/Source/Identifiers.swift

```diff
@@ -33,7 +33,7 @@ import Foundation
 /**
  Communication identifier for Communication Services Users
  */
-@objcMembers public class CommunicationUserIdentifier: NSObject, CommunicationIdentifier {
+@objcMembers public class CommunicationUserIdentifier: NSObject, Codable, CommunicationIdentifier {
```

Comment:
@sacheu this introduction of `Codable` careful need to take into consideration for https://github.com/Azure/azure-sdk-for-ios/pull/655

Reply:
I think I can actually revert this, when encoding/decoding the custom models not actually encoding/decoding CommunicationUserIdentifier